Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -213,6 +215,50 @@ public final class CoreDataManager: StorageManagerType {
             return debugMessages
         }
     }
+
+    /// Note that we have to enumerate all the files in our sandbox to get these properties, so this takes some time.
+    /// It is intended _only_ for use when logging a crash, or other high-value log which won't get in the user's way.
+    private static func storageSizeLogProperties() -> [String: Int64]? {
```

Comment:
A nice thing for readability would be to return `[String: String]` and value as "12.34MB", for example. With these core data storage crashes, I imagine we'll be peaking through multiple Sentry reports and seeing if we see some outliers with thousands of MBs of data. We don't care that much about precision. WDYT?

Reply:
Yep, agreed. Done in 785d6d6

![CleanShot 2024-09-20 at 13 55 06@2x](https://github.com/user-attachments/assets/8ce6f464-6616-483f-b1e8-78334f2f8cd1)

Reply:
Amazing, thanks!